lib/Mount/GroupFolderStorage.php

```diff
@@ -32,18 +33,27 @@ class GroupFolderStorage extends Quota {
 	/** @var ICacheEntry */
 	private $rootEntry;
 
+	/** @var IUserSession */
+	private $userSession;
+
 	public $cache;
 
 	public function __construct($parameters) {
 		parent::__construct($parameters);
 		$this->folderId = $parameters['folder_id'];
 		$this->rootEntry = $parameters['rootCacheEntry'];
+		$this->userSession = $parameters['userSession'];
 	}
 
 	public function getFolderId() {
 		return $this->folderId;
 	}
 
+	public function getOwner($path) {
+		$user = $this->userSession->getUser();
+		return $user !== null ? $user->getUID() : false;
```

Comment:
false is returned here to keep the behavior aligned with the current common implementation. 

When false is returned getOwner on the Node object returns an IUser object with empty uid, which is assumed at some places where getOwner is not checked for null. While not nice, this avoids breaking other existing stuff that calls getOwner